README.md

```diff
@@ -1,2 +1,78 @@
 # archived_interaction
+
 Ruby library to archive http interactions into a git repository for future retrieval
+
+## Usage
+
+### Storing an interaction
+
+The minimum required to store an interaction is a `url` and a `body`.
+
+```ruby
+require 'archived_interaction'
+require 'open-uri'
+
+response = open('http://example.com')
+ArchivedInteraction.new(url: response.base_url, body: response.read).store
+```
+
+### Storing an interaction with a custom timestamp
+
+If you've got an http interaction that happened at some point in the past then
+you can store that with a custom timestamp.
+
+```ruby
+require 'archived_interaction'
+require 'open-uri'
+
+response = open('http://example.com')
+ArchivedInteraction.new(
+  url: response.base_url,
+  body: response.read,
+  timestamp: Time.parse('2016-10-26 15:30')
```

Comment:
does this actually need to be a Time object? If not, probably better to explicitly cast that to `to_s` or whatever is needed, to make it clearer what should be passed in.